guide/source/2.13-migration.md

```diff
@@ -7,8 +7,11 @@ Most of the new features in Meteor 2.13 are either applied directly behind the
 scenes (in a backwards compatible manner) or are opt-in. For a complete
 breakdown of the changes, please refer to the [changelog](http://docs.meteor.com/changelog.html).
 
-For this release all the changes are for quality of life improvements and
-there are no breaking changes.
+In order to correctly run projects in Meteor 2.13 with Docker, you will need to
+update your Dockerfile to use our [Docker image](https://hub.docker.com/r/meteor/node) that contains Nodejs v14.21.4.
+
+For deployments case you are using [Meteor Cloud](https://www.meteor.com/cloud), you don't need to do anything.
+
 
 for more information on the changes in this release, please refer to the
```

Comment:
The changelog with all the changes was already mentioned in the first paragraph. Perhaps this one can be removed.